### PR TITLE
fix: chat top bar iOS

### LINKED ISSUE #16143: Expanded composer is covered by the top bar
Introduced by https://github.com/status-im/status-mobile/pull/15204 cc @alwx

**Steps:** 

1. Open any chat 
2. Fully expand the composer
3. Observe the result

**Actual result:** expanded composer is covered by the top bar

https://github.com/status-im/status-mobile/assets/97245802/9405c390-11b3-45f1-9851-f6371effba2c

### Additional Information

- Status version: nightly 
- Operating System: Android, iOS

[comment]: # (Please, add logs/notes if necessary)

### PR DESCRIPTION
**Pull request by OmarBasem**
fixes: https://github.com/status-im/status-mobile/issues/16143

### CODE REVIEW

File: src/status_im2/contexts/chat/composer/view.cljs [hunk 1]
```diff
@@ -131,7 +132,7 @@
                        :background-y  background-y}
         props         (utils/init-props)
         state         (utils/init-state)]
-    [rn/view
+    [rn/view (when platform/ios? {:style {:z-index 1}})
```
Comment (ibrkhalil) [inline src/status_im2/contexts/chat/composer/view.cljs:135]: I think z-index is only supported on iOS, So we can just use it as it is without conditional.

**Review by ibrkhalil — APPROVED** [review]
Non blocking comment

File: src/status_im2/contexts/chat/composer/view.cljs (hunk 1 above)
Comment (OmarBasem) [reply src/status_im2/contexts/chat/composer/view.cljs:135]: `z-index` is supported on iOS and Android. I want to update the z-index of iOS only here.

**Comment by OmarBasem** [comment]
Hi @pavloburykh, PR is ready for testing

**Review by flexsurfer — APPROVED** [review]

**Review by alwx — APPROVED** [review]

**Comment by alwx** [comment]
I wonder why it's not needed on Android but needed on iOS.

**Comment by OmarBasem** [comment]
> I wonder why it's not needed on Android but needed on iOS.

Are you using different z-indexes for the top bar on Android and iOS?

**Comment by pavloburykh** [comment]
@OmarBasem thanx a lot for the fix! Let's merge it. Failed e2e are not PR related.

**Comment by alwx** [comment]
@OmarBasem no, I don't. Weird that it seems to work differently anyway.

**Pull request merged**